pages/input.html

```diff
@@ -0,0 +1,312 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Posh UI</title>
+
+    <!-- favicon -->
+    <link
+      rel="apple-touch-icon"
+      sizes="180x180"
+      href="assets/brand-logos/apple-touch-icon.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="assets/brand-logos/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="assets/brand-logos/favicon-16x16.png"
+    />
+
+    <!-- font awesome -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+      integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+
+    <!-- highlight.js cdn -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/default.min.css"
+    />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/highlight.min.js"></script>
+
+    <!-- poshui-components cdn -->
+    <link
+      rel="stylesheet"
+      href="https://poshui-components.netlify.app/css/main.css"
+    />
+
+    <!-- poshui-documentation css -->
+    <link rel="stylesheet" href="../app.css" />
+  </head>
+  <body>
+    <div class="container documentation margin-auto padding-2">
+
+      <!-- Header Documentation Starts -->
+
+      <header class="header header-documentation">
+        <img
+          class="img-brand-logo"
+          src="../assets/brand-logos/brand-logo.png"
+          alt="brand logo png"
+        />
+        <a class="link-brand margin-lr-1" href="../index.html">Posh UI</a>
+
+        <nav class="margin-l-auto">
+          <ul>
+            <li class="d-i-block margin-lr-1">
+              <a class="link" href="../index.html">Home</a>
+            </li>
+            <li class="d-i-block margin-lr-1">
+              <a class="link" href="introduction.html">Documentation</a>
+            </li>
+            <li class="d-i-block margin-lr-1">
+              <a class="link"
+                href="https://github.com/hsnice16/PoshUI-Documentation"
+                target="_blank"
+                >GitHub</a
+              >
```

Comment:
just a typo i think. Please format these tags closing tag should be in previous line

Reply:
Actually, I tried to do this but prettier moves it onto the next line.

Reply:
No issues everything else is good. You can go ahead.